Makefile

```diff
@@ -9,7 +9,7 @@ CORE_REPO?=https://github.com/stellar/stellar-core.git
 CORE_REF?=master
 CORE_CONFIGURE_FLAGS?=--disable-tests
 STELLAR_RPC_REF?=main
-HORIZON_REF?=$(shell ./scripts/soroban_repo_to_horizon_repo.sh $(STELLAR_RPC_REF))
```

Comment:
I'm not super familiar with RPC/Horizon relationships, why did we get Horizon branch depending on the RPC branch before?